### PR TITLE
removed all target=blank from base.html

### PR DESCRIPTION
**Pull request by catherinefromont**
Fixes: #210 

**TODOs (if any)**:
**A picture of a cute animal (not mandatory but encouraged)**:

### CODE REVIEW

**Comment by anubhakushwaha** [comment]
@catherinefromont Please add the issue number it fixes.

File: oshc/main/templates/base.html [hunk 1]
```diff
@@ -28,13 +28,13 @@
           <li class="dropdown">
             <a href="#" class="dropdown-toggle" data-toggle="dropdown">Sessions <span class="caret"></span></a>
             <ul class="dropdown-menu" role="menu">
-              <li><a href="https://www.youtube.com/channel/UC1_IAby-9me3iICtxuiDL5Q/videos" target="_blank">View Session</a></li>
+              <li><a href="https://www.youtube.com/channel/UC1_IAby-9me3iICtxuiDL5Q/videos">View Session</a></li>
```
Comment (amarlearning) [inline oshc/main/templates/base.html]: undo this change

**Comment by amarlearning** [comment]
Thanks, @catherinefromont :100: But it looks like this issue has been resolved only the changes are pending and needed to be migrated to the master branch. 

@tapasweni-pathak reminder.

**Review by tapaswenipathak — COMMENTED** [review]
@catherinefromont These URLs need to be open in new tab, only which are a relative link/paths should be loaded in same tab. I need to update the master branch.
